mp2-v1/src/values_extraction/mod.rs

```diff
@@ -45,6 +48,15 @@ pub(crate) const OUTER_KEY_ID_PREFIX: &[u8] = b"\0OUT_KEY";
 
 pub(crate) const BLOCK_ID_DST: &[u8] = b"BLOCK_NUMBER";
 
+/// Compute the identifier for a column of a table containing off-chain data
+pub fn identifier_offchain_column(table_name: &str, column_name: &str) -> u64 {
+    let inputs: Vec<F> = vec![table_name, column_name]
```

Comment:
should we put a DST just because ? like `OFFCHAIN || table_name || col_name` ?

Reply:
Done in commit https://github.com/Lagrange-Labs/mapreduce-plonky2/pull/415/commits/4a513aa8feffef4d226783367d061a59d56ecdd8